cmake/developer_package/packaging.cmake

```diff
@@ -53,7 +53,9 @@ macro(ie_cpack)
     set(CPACK_PACKAGE_VENDOR "Intel Corporation")
     set(CPACK_VERBATIM_VARIABLES ON)
     set(CPACK_COMPONENTS_ALL ${ARGN})
-    set(CPACK_STRIP_FILES ON)
+    if (NOT CPACK_STRIP_FILES)
```

Comment:
`if (NOT DEFINED CPACK_STRIP_FILES)`

Otherwise, `cmake -DCPACK_STRIP_FILES=OFF` still sets option to `ON`